code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -1,6 +1,6 @@
 /datum/species/jelly
 	// Entirely alien beings that seem to be made entirely out of gel. They have three eyes and a skeleton visible within them.
-	name = "Xenobiological Jelly Entity"
+	name = "Jellyperson"

```

Comment:
Seems completely unrelated.

Reply:
name on the scanner stuff prolly

Reply:
Yeah, i use the species.name for the display name, and this didn't matter before since it was invisible ingame

Reply:
I actually like the first one better lol

Reply:
It is nice but base jellypeople are no longer a thing, so it's better to have a catch-all name for jelly species in this case.